MBTI Quiz/dist/index.html

```diff
@@ -44,357 +38,353 @@
 
   </nav>
 
-  <!-- Card -->
-
-  <!-- Card -->
-
-
-
   <section id = "top" class = "question1" >
-      <br><br><br><br>
-      <div class="card text-center container-small shadow-lg">
+    <br><br><br><br>
+    <div class="card text-center container-small shadow-lg">
+
 
-        <!-- Card content -->
       <div class="card-body">
 
-        <!-- Title -->
+
         <h2 class="card-title"><a><strong>Myers Briggs Personality Identifier</strong></a></h2><br><br>
-        <!-- Text -->
+
         <h5 class="card-text">Try answer as many questions you can to help identify the persons myers briggs type. </h5>
         <h5 class="card-text">If you don't know the answer, skip it and move on to the next. </h5><br>
       </div>
 
-      </div>
+    </div>
 
-      <br><br>
+    <br><br>
```

Comment:
Remove

Reply:
removed most of these but had to keep some as on the traits and careers and results cards even with margin-top on them when scrolling to those sections it would but off a section of the card without the br's